index.html

```diff
@@ -19,20 +19,20 @@ <h1 id="title"> Microverse Survey Form</h1>
           <p id="description">Thank you for taking the time to help us improve the platform</p>
       </div>
```

Comment:
You can use the `header` tag, instead of using the `div` block with the 'header' class

Reply:
I think you should also give reasons **WHY** you're encouraging them to use the semantic HTML element. This will give the student more insight and conviction on making the requested changes.